tox.ini

```diff
@@ -39,6 +39,7 @@ deps=
     pytest-randomly==3.11.0
     pytest-cov==3.0.0
     pytest-asyncio==0.18.0
+    pytest-order==1.0.1
```

Comment:
I don't think adding one more dependency just for this one scenario is a really good idea